activity2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<table>
+    <row1>
+        <head>Empno</head>
+        <head>Empname</head>
+        <head>Salary</head>
+    </row1>
```

Comment:
should have used these for element names instead of making separate row 
```
<row>
    <empno></empno>
    <empname></empname>
    <salary></salary>
</row>
```